console/src/main/java/org/eclipse/kapua/app/console/client/data/TopicsTable.java

```diff
@@ -114,7 +114,7 @@ public void onSuccess(List<GwtTopic> topics) {
             public void onFailure(Throwable t) {
                 FailureHandler.handle(t);
                 topicInfoGrid.unmask();
-                Window.alert(MSGS.noDataMessage());
+                Window.alert((String)MSGS.noDataMessage());
```

Comment:
Can you explain why this is necessary. Because it will be removed again when the Kapua cleanup profile will be applied.